src/App.css

```diff
@@ -26,3 +26,9 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+.movieitem-wrapper {
```

Comment:
This CSS is related to different component. To keep component using this css class work anywhere in the application it's better to separate it's css into it's own file and keep it next to js file containing this component and just import it there.